lib/nx_zlib.h

```diff
@@ -525,7 +525,7 @@ extern uint64_t nx_wait_ticks(uint64_t ticks, uint64_t accumulated_ticks, int do
 
 /* zlib crc32.c and adler32.c */
 extern unsigned long nx_crc32_combine(unsigned long crc1, unsigned long crc2, uint64_t len2);
-extern unsigned long nx_adler32_combine(unsigned long adler1, unsigned long adler2, uint64_t len2);
+extern unsigned long nx_adler32_combine(unsigned long adler1, unsigned long adler2, off_t len2);
```

Comment:
You can't change this declaration without changing `lib/nx_adler_combine.c` too.
The function is still treating len2 as `uint64_t`.